configs/common/packages.yaml

```diff
@@ -143,9 +154,9 @@ packages:
     require: '@2.46.3 ~shared ~f2py'
     variants: '+pflogger'
   # To avoid duplicate packages being built
-  meson:
-    require:
-    - '@1.5.1:'
+  #meson:
```

Comment:
Note to self. Remove all these commented-out packages once we are sure it all works as expected